packages/javascript/bh-shared-ui/src/mocks/handlers/searchHandlers.ts

```
@@ -0,0 +1,53 @@
+
+import { rest } from 'msw';
+import { createMockSearchResults } from '../factories/initial';
+
+export const searchHandlers = [
+    rest.get('/api/v2/search', (req, res, ctx) => {
+        const nodeType = req.url.searchParams.get('type');
+        return res(
+            ctx.json({
+                data: Array(10)
+                    .fill(null)
+                    .map(() => {
```

Biome (2.3.13)

[error] This callback passed to map() iterable method should always return a value.

Add missing return statements so that this callback returns a value on all execution paths.

(lint/suspicious/useIterableCallbackReturn)